src/config/system_stats.rs

```diff
@@ -0,0 +1,123 @@
+use lazy_static::lazy_static;
+#[cfg(target_os = "macos")]
+use regex::Regex;
```

Comment:
Can we move this conditional `use`into the function that uses it?(just `use` in get_battery_capacity, rather than here)